core/src/lib/code-generation/code-builder/queued-code-builder.ts

```diff
@@ -145,6 +150,31 @@ export class QueuedCodeBuilder{
         return this;
     }
 
+    /**
+     * add an import to the CodeBuilder Instance.
+     *
+     * @param {string} element which element should be imported. ```import {<Element>} from '...'```
+     * @param {string} from which library should imported. ```import {...} from '<From>'```
+     * @param {string} [importAs] use alias for this element. ```import {<Element> as <importAs>} from '...'```
+     * @returns {QueuedCodeBuilder}
+     * @memberof QueuedCodeBuilder
+     */
+    addImport(element: string, from: string, importAs?: string): QueuedCodeBuilder {
+        this.imports.add(element, from, importAs);
+        return this;
+    }
+
+    /**
+     * determines the location where all imports from the CodeBuilder instance should be placed.
+     *
+     * @returns {QueuedCodeBuilder}
+     * @memberof QueuedCodeBuilder
+     */
+    addImportStatements(): QueuedCodeBuilder {
```

Comment:
addImportStatementToQueue()

Reply:
there is no function called add to queue, it means, that the import statements will add an this building point in the code...